### DISCUSSION TITLE
Dark, Light, and Pink Wallpapers

### DISCUSSION THREAD

**Discussion by fernbacher** [Atlas-OS/Atlas#109] [Community Artwork]
![Atlas_Windows_Fern](https://user-images.githubusercontent.com/66071319/154816232-65d2757f-8c2c-4e38-b1ec-b7d69e799a95.png)
![Atlas_WindowsDark_Fern](https://user-images.githubusercontent.com/66071319/154816234-e69012ba-74ce-430e-a3fe-2a1da507710c.png)
![wall2_atlas](https://user-images.githubusercontent.com/66071319/154819217-08dbc81c-c4c3-4c39-8542-e0bad7ef17a4.png)
Template: https://wallhaven.cc/w/o3vjvm
Template2: https://wallhaven.cc/w/e7p2pk

**Comment by JayK-Github** [comment]
yoink, thanks for the [sick ](https://www.urbandictionary.com/define.php?term=that%27s%20sick)wallpaper

**Comment by he3als** [comment]
the first two are amazing
